CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Below is a list of notable changes implemented in this repository.
 
-## [v0.10.0](https://github.com/OpenSTFoundation/brandedtoken-contracts/releases/tag/v0.10.0) <release date: DD MM YYYY>
+## 0.10.0 <release date: DD MM YYYY>
```

Comment:
I have added the same on bt.js release PR after referring to mosaic.js PR. Is the release tag url not needed? I may be missing bigger picture.
https://github.com/OpenSTFoundation/brandedtoken.js/pull/78/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR5
https://github.com/OpenSTFoundation/mosaic.js/pull/101/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR5

Reply:
Fair question. I added the link there when I had a different understanding of the process and I don't think it is good UX to have a non-functional link for an extended period of time.

And good point. It did not occur to me to comment it out, which is a good solution for marking the place. I will push a change!

Reply:
Pushed! Though with two departures:
- struck the word "Release" as that should be true for all
- specified the date format

Reply:
I will update the same on my PR.